tests/components/blebox/test_config_flow.py

```diff
@@ -0,0 +1,181 @@
+"""Test Home Assistant config flow for BleBox devices."""
+
+from asynctest import CoroutineMock, PropertyMock, mock, patch
+import blebox_uniapi
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.blebox import config_flow
+from homeassistant.setup import async_setup_component
+
+from .conftest import mock_config, mock_only_feature, setup_product_mock
+
+
+@pytest.fixture
+def flow(hass):
+    """Return a user configuration flow."""
```

Comment:
@MartinHjelmare Is this ok or does he need to call `async_init` instead?

```
    result = await hass.config_entries.flow.async_init(
        DOMAIN, context={"source": config_entries.SOURCE_USER}
    )
```

Reply:
Fixed and refactored, thanks!